API.yaml

```
@@ -1,548 +1,670 @@
+  description: |-
+    The Exposure Reporting Service makes the TEK Chunks created by the
+    Exposure Ingestion Service available to the Mobile Client. Only TEK Chunks for
+    the last 14 days are made available. To avoid downloading the same TEKs multiple
+    times, the Mobile Clients fetch the indexes of the TEK Chunks available to download
+    first. Then, they only actually download TEK Chunks with indexes for which TEK
+    Chunks have not been downloaded before.
+- name: Analytics
+  description: |-
+    The Analytics Service provides an API to the Mobile Clients for uploading
+    certain data without identifying users, both during regular operations and especially
+    when a match is found between a TEK Chunk and the RPIs in the RPI Database. Collecting
+    these data is crucial to spotting anomalies in the system, as well as being able
+    to check how many users are being notified. The National Healthcare System needs
+    this information to operate Immuni effectively.
 paths:
   /settings:
     get:
       tags:
-      - "App Configuration"
-      summary: "The Mobile Client fetches the Configuration Settings. Different Configuration Settings may be made available for different platforms (i.e., iOS and Android) and App build numbers."
-      produces:
-      - "application/json"
+      - App Configuration
+      summary: >-
+        The Mobile Client fetches the Configuration Settings. Different Configuration
+        Settings may be made available for different platforms (i.e., iOS and Android)
+        and App build numbers.
       parameters:
-      - name: "platform"
-        in: "query"
-        description: "The Mobile Client's platform."
+      - name: platform
+        in: query
+        description: The Mobile Client's platform.
         required: true
-        type: "string"
-        enum:
-        - "ios"
-        - "android"
-      - name: "build"
-        in: "query"
-        description: "The App's build number."
+        schema:
+          type: string
+          enum:
+          - ios
+          - android
+      - name: build
+        in: query
+        description: The App's build number.
         required: true
-        type: "integer"
-        minimum: 1
-        format: "int32"
+        schema:
+          minimum: 1
+          type: integer
+          format: int32
       responses:
         "200":
-          description: "A JSON-formatted dictionary. Specifications are being worked out."
+          description: A JSON-formatted dictionary. Specifications are being worked
+            out.
           headers:
             Cache-Control:
-              type: "string"
-              description: "Directives for caching the response. The Configuration Settings are cached for 60 minutes."
+              description: Directives for caching the response. The Configuration
+                Settings are cached for 60 minutes.
+              schema:
+                type: string
+          content: {}
```

I think something like that. 

The converter didn't add the media-type because there was no content specified in the spec.

```suggestion
          content:
            application/json:
              schema:
                type: object
```

---

I'd probably go for this second version you proposed. I prefer it because it's more explicit as it was before [here](https://github.com/immuni-app/immuni-documentation/pull/33/files#diff-4ebe1e340e131605718b39232a213acfL25). What's your take?

---

@ocardia just commit the suggestion then :)

---

I got no write permission, so I can't accept the suggestion. 😞